lib/operations/result.rb

```diff
@@ -113,7 +113,7 @@ def errors_with_code?(name, *names)
 
   def context_as_json
     context.transform_values do |context_value|
-      next context_value.class.name unless context_value.respond_to?(:id)
+      next context_value.inspect unless context_value.respond_to?(:id)
```

Comment:
@pyromaniac just thinking, won't this produce very extended output, possibly having sensitive attributes from context which could be reported to sentry/logging?

Reply:
It could, however, AR/entities/aggregates cases are covered with a condition and I frankly don't see more object types containing sensitive data. Meanwhile, the previous implementation gave us nothing in terms of problem understanding